### PR TITLE
Doc for API AM role

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Doc for API AM role

### Resolves:
[OKTA-107617](https://oktainc.atlassian.net/browse/OKTA-107617)

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
hosed branch again. @jmelberg-okta Can you give me any facetime tomorrow? I can't get a clean doc PR.

File: _source/_use_cases/api_access_management/index.md [hunk 1]
```diff
@@ -103,12 +103,22 @@ Okta API Products refers to all the resources and tools that Okta makes availabl
...
 * [Resource (API) Servers](#resource-api-servers)
 
+#### API Access Management Administrator Role
+
+Okta provides the API Access Management Administrator role to make managing authorization servers. A user with this role can perform the following tasks:
+
+* Create and edit authorization servers, scopes, custom claims, and access policies
+* Create and edit  OAuth and OIDC client apps
+* Assign users and groups to OAuth/OIDC client apps
+* View user profiles so that you can assign users or clients in the Token Preview dialog
```
Comment (danielzeller-okta) [inline _source/_use_cases/api_access_management/index.md]: No need to call out why you can view user properties here.

**Comment by mystiberry-okta** [comment]
DO NOT MERGE!!!!!

File: _source/_use_cases/api_access_management/index.md (hunk 1 above)
Comment (mystiberry-okta) [reply _source/_use_cases/api_access_management/index.md]: will delete.

**Comment by mystiberry-okta** [comment]
Waiting for code jira to merge next week. Do not merge.

**Comment by mystiberry-okta** [comment]
Dev approved in slack.

**Pull request merged**